contributors/design-proposals/job.md

```diff
@@ -29,27 +29,31 @@ here would be the ability to implement any type of batch oriented tasks.
 
 ## Backoff policy and failed pod limit
 
-By design, Jobs do not have any notion of failure, other than Pod's `restartPolicy`
+By design, Jobs do not have any notion of failure, other than a pod's `restartPolicy`
 which is mistakenly taken as Job's restart policy ([#30243](https://github.com/kubernetes/kubernetes/issues/30243),
 [#[43964](https://github.com/kubernetes/kubernetes/issues/43964)]).  There are
-situation where one wants to fail a Job after some amount of retries over certain
+situation where one wants to fail a Job after some amount of retries over a certain
 period of time, due to a logical error in configuration etc.  To do so we are going
-following fields will be introduced, which will control the exponential backoff
-when retrying Job: number of retries and time to retry.  The two fields will allow
-creating a fine grain control over the backoff policy, limiting the number of retries
-over specified period of time.  In the case when only one of them is specified
-an exponential backoff with duration of 10 seconds and factor of 2 will be applied
-in such a way that either time or number is reached.  After reaching the limit
-a Job will be marked as failed.
-
-Additionally, to help debug the issue with a job, and limit the impact of having
-too many failed pods left around (as mentioned in [#30243](https://github.com/kubernetes/kubernetes/issues/30243))
+to introduce following fields will be introduced, which will control the exponential
+backoff when retrying a Job: number of retries and time to retry.  The two fields
+will allow creating a fine-grained control over the backoff policy, limiting the
```

Comment:
- [ ] Reading this again, I think this would read better as  
  > will allow fine-grained control over